mapiproxy/servers/default/emsmdb/oxomsg.c

```diff
@@ -422,6 +422,7 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopTransportSend(TALLOC_CTX *mem_ctx,
 		DEBUG(0, ("Not implemented yet - shouldn't occur\n"));
 		break;
 	case true:
+		emsmdbp_object_attach_sharing_metadata_XML_file(emsmdbp_ctx, object);
```

Comment:
This will generate warning with clang. Check retval and add debug message upon failure or return void.